ha/k8s_setup/ha_setup.py

```diff
@@ -391,6 +404,12 @@ def _add_node_health(self, data_node_ids, server_node_ids, control_node_ids) ->
                                    resource_type=CLUSTER_ELEMENTS.NODE.value,
                                    resource_id=node_id,
                                    specific_info=specific_info)
+        if node_id_list:
```

Comment:
This can be re-written as `if node_id_list  and kv_enable_batch_put:`

Reply:
Yes. Thanks. instead of a list, I am using another flag now. So, checking that flag here.